modules/@apostrophecms/doc-type/index.js

```diff
@@ -1222,7 +1222,7 @@ module.exports = {
         }
 
         const { aposShareKey: _aposShareKey, ...draft } = doc;
-        const aposShareKey = doc.aposShareKey || self.apos.util.generateId();
+        const aposShareKey = _aposShareKey || self.apos.util.generateId();
```

Comment:
ℹ️ just a minor change, so we benefit from the `_aposShareKey` variable declaration, not just leaving it to its sole "property extraction" purpose

Reply:
By convention `_` is used for unused variables, you have just renamed it in this case.
That's a detail of course

Reply:
oh okay, I would have though it was for private variables, or to find a way to rename a variable in a simple way.
I'll revert this line